.gitlab-ci.yml

```diff
@@ -81,26 +73,21 @@ stages:
 
 .build_blueos_3_ppc64le_ib_p9_script:
   script:
-    #BUILD
-    - echo -e "Build ${CI_PROJECT_NAME}"
-    - lalloc 1 -W 5 ${LSF_ACCOUNT} scripts/gitlab/build_and_test.sh --build-only
-    - echo -e "section_end:$(date +%s):build\r\e[0K"
-    #TEST
-    - echo -e "Test ${CI_PROJECT_NAME}"
-    - lalloc 1 -W 5 ${LSF_ACCOUNT} scripts/gitlab/build_and_test.sh --test-only
+    #BUILD + TEST
+    - echo -e "Build and test ${CI_PROJECT_NAME}"
+    - lalloc 1 -W 10 ${LSF_ACCOUNT} scripts/llnl/build_src.py --host-config ${HOST_CONFIG}
 
 .build_with_deps_blueos_3_ppc64le_ib_p9_script:
   script:
     - *run_update_uberenv
     #DEPENDENCIES
-    - echo -e "Build dependencies and generate host-config file (uberenv)"
-    - lalloc 1 -W 20 ${LSF_ACCOUNT} scripts/gitlab/build_and_test.sh --deps-only
-    #BUILD
-    - echo -e "Build ${CI_PROJECT_NAME}"
-    - lalloc 1 -W 5 ${LSF_ACCOUNT} scripts/gitlab/build_and_test.sh --build-only
-    #TEST
-    - echo -e "Test ${CI_PROJECT_NAME}"
-    - lalloc 1 -W 5 ${LSF_ACCOUNT} scripts/gitlab/build_and_test.sh --test-only
+    - echo -e "section_start:$(date +%s):dependencies\r\e[0K
+      Build dependencies and generate host-config file (uberenv)"
+    - lalloc 1 -W 20 ${LSF_ACCOUNT} scripts/llnl/build_tpls.py --spec=${SPEC} --directory=${CI_PROJECT_NAME}
+    - echo -e "section_end:$(date +%s):dependencies\r\e[0K"
+    #BUILD + TEST
+    - echo -e "Build and test ${CI_PROJECT_NAME}"
+    - lalloc 1 -W 10 ${LSF_ACCOUNT} scripts/llnl/build_src.py
```

Comment:
should probably pass the host-config on this line though i guess there is only one at this point,.